### PR TITLE
[naoqi_apps] add node and launch for ALGazeAnalysis

### PR DESCRIPTION
**Pull request by kochigami**
I followed (http://doc.aldebaran.com/2-4/naoqi/peopleperception/algazeanalysis-api.html#algazeanalysis-api) and this requires https://github.com/ros-naoqi/naoqi_bridge_msgs/pull/25

If there is any problem, please let me know.

services:
- /set_gaze_analysis_tolerance
  is for `ALGazeAnalysisProxy::setTolerance`
  and uses SetGazaAnalysisTolerance.srv
- /get_gaze_analysis_tolerance
  `ALGazeAnalysisProxy::getTolerance`
  uses GetGazaAnalysisTolerance.srv

topics: 
- /gaze_analysis
  is for 

```
    PeoplePerception/Person/<ID>/EyeOpeningDegree
    PeoplePerception/Person/<ID>/GazeDirection
    PeoplePerception/Person/<ID>/HeadAngles
    PeoplePerception/Person/<ID>/IsLookingAtRobot
    PeoplePerception/Person/<ID>/LookingAtRobotScore
```

and uses GazaAnalysis.msg
- /person_starts_looking_at_robot
  is for `GazeAnalysis/PersonStartsLookingAtRobot()`
  (and uses Int64.msg)
-  /person_stops_looking_at_robot
  is for `GazeAnalysis/PersonStopsLookingAtRobot()`
  (and uses Int64.msg)
- /people_looking_at_robot
  is for `GazeAnalysis/PeopleLookingAtRobot()`
  (and uses Int64MultiArray.msg)

### CODE REVIEW

**Comment by kochigami** [comment]
- delete `self.run()` which publishes topics => I separated the function to https://github.com/ros-naoqi/naoqi_bridge/pull/89
- keep services, but used srv files are replaced from `naoqi_bridge_msgs` (https://github.com/ros-naoqi/naoqi_bridge_msgs/pull/25) to `std_srvs` and `naoqi_bridge_msgs`  (https://github.com/ros-naoqi/naoqi_bridge_msgs/pull/37)   
- This requires https://github.com/ros-naoqi/naoqi_bridge_msgs/pull/37.

I just tested with Pepper (Naoqi 2.5.5.5)+ ROS indigo.

File: naoqi_apps/nodes/naoqi_gaze_analysis.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        NaoqiNode.__init__(self, 'naoqi_gaze_analysis')
+        self.connectNaoQi()
+        self.setToleranceSrv = rospy.Service("set_gaze_analysis_tolerance", SetFloat, self.handleSetGazeAnalysisTolerance)
+        self.getToleranceSrv = rospy.Service("get_gaze_analysis_tolerance", GetFloat, self.handleGetGazeAnalysisTolerance)
+        rospy.loginfo("naoqi_gaze_analysis is initialized")
+
+    def connectNaoQi(self):
+        rospy.loginfo("Connecting to NaoQi at %s:%d", self.pip, self.pport)
+        self.gazeProxy = self.get_proxy("ALGazeAnalysis")
+        if self.gazeProxy is None:
```
Comment (nlyubova) [inline naoqi_apps/nodes/naoqi_gaze_analysis.py]: Could you please add an error log/message here ? Like cannot get GazeAnalysis proxy?

Comment (kochigami) [reply naoqi_apps/nodes/naoqi_gaze_analysis.py]: Thank you very much for your review.
I added an error log/ message.

I will try this with NAO. 
When I finish testing all the conditions below, I will write a comment.
- [x] Pepper + ros indigo 
- [ ] Pepper + ros kinetic
- [ ] NAO + ros indigo
- [ ] NAO + ros kinetic

**Comment by nlyubova** [comment]
Thank you so much for your work!

**Comment by kochigami** [comment]
I tested this with 
- [x] Pepper (NAOqi 2.5.5.5) + ROS indigo
- [x] Pepper (NAOqi 2.5.5.5) + ROS kinetic
- [x] NAO (NAOqi 2.1.4) + ROS indigo
- [x] NAO (NAOqi 2.1.4) + ROS kinetic

**Comment by nlyubova** [comment]
Thank you so much for testing!

**Pull request merged**